### ISSUE TITLE
do not return queryString in PDO::query()!!

### ISSUE THREAD

**Issue by parsibox** [taq/pdooci#39]
hi
very bad problem
in function PDO::query()  you do not return queryString!

**Comment by taq** [comment]
I'm missing your point. PHP `PDO::query` accepts a SQL string and returns a statement:

https://www.php.net/manual/en/pdo.query.php

```
Parameters
query
The SQL statement to prepare and execute.

Return Values
Returns a PDOStatement object or false on failure.
```

That's exactly what `PDO::query` does here. They are compatible. What is wrong? It's not supposed to return the query string.

**Comment by taq** [comment]
Not really.

Taliking about the original `PDO`, we can get the _query string_ from the _statement_ using 

```
$statement->queryString;
```

Check https://www.php.net/manual/en/class.pdostatement.php :

```
class PDOStatement implements Traversable {
/* Properties */
readonly string $queryString;
```

So, if you use 

```
$query_id->getStatement();
```

as said, it will return _the statement_, not the _SQL string_, which is correct.

To the get the SQL string, now it's a new implementation, because we don't have the `$queryString` property on the `Statement` class and it will need to be implemented as a readonly property and stored on the `Statement` constructor.

Is that correct? But I wouldn't use readonly properties, because it's only available on PHP 8.1 and I don't want to break compatibility with older versions.
